frontend/src/queries/nodes/DataTable/ColumnConfigurator/ColumnConfigurator.tsx

```diff
@@ -66,8 +68,8 @@ function ColumnConfiguratorModal(): JSX.Element {
     // setting the container to be larger than we need
     // and adding a container with a smaller height to each row item
     // allows the new row item to set a margin around itself
-    const rowContainerHeight = 36
-    const rowItemHeight = 32
+    const rowContainerHeight = 80
+    const rowItemHeight = 75
```

Comment:
Suspicious odd number 👀 Also, these rows are now very large. Why?

Reply:
yeah, this entire modal needs to change. I think it's worthy of its own PR, but I'll see if I can make it just a bit better.